extensions/wikia/UserLogin/WikiaConfirmEmailSpecialController.class.php

```diff
@@ -78,7 +78,7 @@ public function index() {
 			}
 
 			$expUser = User::newFromConfirmationCode( $this->code );
```

Comment:
There are two other instances where the output of this method is not verified properly in SpecialConfirmemail ([L#161](https://github.com/Wikia/app/blob/dev/includes/specials/SpecialConfirmemail.php#L161) & [L#208](https://github.com/Wikia/app/blob/dev/includes/specials/SpecialConfirmemail.php#L208)). Should we fix those too while at it?